terraform/cf-certs/upload.tf

```diff
@@ -7,6 +7,7 @@ resource "aws_iam_server_certificate" "system" {
   certificate_body  = "${var.system_domain_crt}"
   private_key       = "${var.system_domain_key}"
   certificate_chain = "${var.system_domain_intermediate_crt}"
+  path              = "/cloudfront/${var.env}-system-domain"
```

Comment:
As suggested in the IRL conversation, I have created a new commit in this PR. Please, let me know, if that still prevents the pipeline from progression.